### PR TITLE
chore: Tablette chocolat noir

### PR DESCRIPTION
**Pull request by moon-rabbitOFF**
### What
- Improvements to the Categories taxonomy
### Part of
- #5507

### CODE REVIEW

**Comment by stephanegigandet** [comment]
It looks like there are a lot of conflicts that have not been resolved:

<<<<<<< Updated upstream

**Comment by moon-rabbitOFF** [comment]
@stephanegigandet Is it ok now?

**Comment by VaiTon** [comment]
@moon-rabbitOFF still conflicts.

**Comment by moon-rabbitOFF** [comment]
It should be ok now

**Comment by svensven** [comment]
@moon-rabbitOFF I had a look at resolving the conflict, but so much has changed in categories.txt since then, it may be simpler just to start a new edit with your changes.

File: taxonomies/categories.txt [hunk 1]
```diff
@@ -98310,74 +98406,1356 @@ en:miscellaneous pet food, miscellaneous animal food
...
+fr:Brochettes, brochette
+nl:Spiesjes
+
+en:edible tableware
+de:essbares Geschirr
+fr:Couverts comestibles, Vaisselles comestibles
 
-<en:Proposed for deletion
-en:Empty, Actually empty, No data
-ru:Пустое, Пустые
```
Comment (VaiTon) [inline taxonomies/categories.txt:98383]: I don't think that's right

Comment (aleene) [reply taxonomies/categories.txt:98383]: It is not very relevant as it is not related to the product. Wehave the states to track completeness.

**Review by aleene — APPROVED** [review]

**Pull request merged**